.github/workflows/conformance-aks-v1.10.yaml

```diff
@@ -136,6 +136,16 @@ jobs:
           fi
 
           CILIUM_INSTALL_DEFAULTS="--cluster-name=${{ env.name }} \
+            --chart-directory=install/kubernetes/cilium \
+            --helm-opts=image.repository=quay.io/cilium/cilium-ci \
+            --helm-opts=image.tag=${SHA} \
+            --helm-opts=operator.image.repository=quay.io/cilium/operator \
+            --helm-opts=operator.image.suffix=-ci \
+            --helm-opts=operator.image.tag=${SHA} \
+            --helm-opts=clustermesh.apiserver.image.repository=quay.io/cilium/clustermesh-apiserver-ci \
+            --helm-opts=clustermesh.apiserver.image.tag=${SHA} \
+            --helm-opts=hubble.relay.image.repository=quay.io/cilium/hubble-relay-ci \
+            --helm-opts=hubble.relay.image.tag=${SHA} \
             --agent-image=quay.io/${{ github.repository_owner }}/cilium-ci \
             --operator-image=quay.io/${{ github.repository_owner }}/operator-azure-ci \
```

Comment:
Isn't this redundant now? Also, shouldn't we use `quay.io/${{ github.repository_owner }}/cilium-ci` in the `helm-opts=image.repository` instead of hard-coding `cilium/cilium`?

(same applies to similar changes below)